hcl-mode.el

```diff
@@ -184,23 +186,30 @@
     map)
   "Keymap for Hcl major mode.")
 
+(defconst hcl-mode-syntax-table
+  (let ((table (make-syntax-table)))
+    (prog1
+        table
+      (modify-syntax-entry ?_ "_" table)
+      (modify-syntax-entry ?- "_" table)
```

Comment:
I think these two are superfluous, since `_` and `-` already have symbol syntax inherited from `prog-mode`, right?

Reply:
Hmm, not sure.

Reply:
Yes, you could be right. I'm fine with making these explicit, though. I've merged this change into my `master`, and my fork is now the official MELPA source for this package (see https://github.com/melpa/melpa/issues/6674).

Reply:
Great, thanks again.